lib/reporting/sp_idv_weekly_dropoff_report.rb

```diff
@@ -0,0 +1,478 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
```

Comment:
seeing as this report needs the prod database to convert UUIDs, it can't run locally. should we yank the command line optoins part?

```suggestion
require 'csv'
require 'reporting/cloudwatch_client'
require 'reporting/cloudwatch_query_quoting'
```

Reply:
This one does not actually need to convert UUIDs! That is the one in #11787. I'm planning to go back to that one and address feedback now that this one is green. I'll remove the command line options there.

Reply:
oh sorry my brain is mush I couldn't remember which PR

Reply:
I cleaned this up on that PR